java/feathub-connectors/flink-connectors/flink-connector-redis/src/main/java/com/alibaba/feathub/flink/connectors/redis/sink/RedisSinkConfigs.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2022 The Feathub Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.feathub.flink.connectors.redis.sink;
+
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.configuration.ConfigOptions;
+
+/** Configurations used by Redis sink. */
+public class RedisSinkConfigs {
+
+    static final ConfigOption<String> HOST =
+            ConfigOptions.key("host")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription("The host of the Redis instance to connect.");
+
+    static final ConfigOption<Integer> PORT =
+            ConfigOptions.key("port")
+                    .intType()
+                    .defaultValue(6379)
+                    .withDescription("The port of the Redis instance to connect.");
+
+    static final ConfigOption<String> PASSWORD =
+            ConfigOptions.key("password")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription("The password of the Redis instance to connect.");
+
+    static final ConfigOption<Integer> DB_NUM =
+            ConfigOptions.key("dbNum")
+                    .intType()
+                    .defaultValue(0)
+                    .withDescription("The No. of the Redis database to connect.");
+
+    static final ConfigOption<String> TABLE_NAME =
+            ConfigOptions.key("tableName")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription(
+                            "A name that uniquely identifies the input table. "
```

Comment:
Why do we need to identify the "input table"?

Maybe we can simply change the semantics of this parameter to represent the Redis table that this sink refers to.